pkgs/servers/mqtt/mosquitto/default.nix

```diff
@@ -30,13 +30,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mosquitto";
-  version = "2.0.18";
+  version = "2.0.20";
 
   src = fetchFromGitHub {
     owner = "eclipse";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-Vs0blV2IhnlEAm0WtOartz+0vLesJfp78FNJCivRxHk=";
+    sha256 = "sha256-oZo6J6mxMC05jJ8RXIunOMB3kptA6FElchKlg4qmuQ8=";
```

Comment:
```suggestion
    hash = "sha256-oZo6J6mxMC05jJ8RXIunOMB3kptA6FElchKlg4qmuQ8=";
```

Reply:
https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub

> hash corresponds to the hash of the extracted directory. Again, other hash algorithms are also available, but hash is currently preferred.

Reply:
sorry docs for fetch from github not very clear but docs for fetch url have

> It is recommended that you use the hash attribute instead of the other hash-specific attributes that exist for backwards compatibility.